features/whitelist.feature

```diff
@@ -0,0 +1,70 @@
+@omniauth
```

Comment:
some inconsistent indenting, and the scenarios are rather imperative - any chance we could evolve them towards a more declarative stance?

* http://itsadeliverything.com/declarative-vs-imperative-gherkin-scenarios-for-cucumber
* https://danashby.co.uk/2014/12/19/declarative-or-imperative-how-do-you-write-your-feature-file-scenarios/

also the feature description should tell us why someone wants this feature e.g.

``` 
As a projectscope admin
So that I can restrict app access to certain people
I would like to have a whitelist of users who can access the app and restrict everyone else
```